frontend/src/metabase/public/containers/PublicDashboard.jsx

```diff
@@ -166,14 +167,19 @@ class PublicDashboard extends Component {
               <DashboardTabs location={this.props.location} />
               <Separator />
               <DashboardGridContainer>
-                <DashboardGridConnected
-                  {...this.props}
-                  isPublic
-                  className="spread"
-                  mode={PublicMode}
-                  metadata={this.props.metadata}
-                  navigateToNewCardFromDashboard={() => {}}
-                />
+                <FixedWidthContainer
+                  data-testid="fixed-width-container"
+                  isFixedWidth={dashboard?.width === "fixed"}
+                >
+                  <DashboardGridConnected
+                    {...this.props}
+                    isPublic
+                    className="spread"
+                    mode={PublicMode}
+                    metadata={this.props.metadata}
+                    navigateToNewCardFromDashboard={() => {}}
+                  />
```

Comment:
@JesseSDevaney WDYT about making `FixedWidthContainer` a part of `DashboardGridConnected`?

We have three top-level dashboard components (`DashboardApp,` `PublicDashboard`, and `AutomaticDashboardApp`), and they all use `DashboardGridConnected`. I think it'd make sense to move the width logic there, so all three dashboard components can get this featyre for free

Reply:
Great call-out. I moved the `FixedWidthContainer` into `DashboardGridConnected`